src/Umbraco.Web.BackOffice/Controllers/UserGroupsController.cs

```diff
@@ -164,7 +166,7 @@ public IEnumerable<UserGroupBasic> GetUserGroups(bool onlyCurrentUserGroups = tr
         /// Return a user group
         /// </summary>
         /// <returns></returns>
-        [UserGroupAuthorization("id")]
+        [Authorize(Policy = AuthorizationPolicies.UserBelongsToUserGroupInRequest)]
```

Comment:
The querystring name is `id` and not `userGroupIds` like below, so out auth is not hit correct

Reply:
Yes, the policy looks for both query strings

```cs
            options.AddPolicy(AuthorizationPolicies.UserBelongsToUserGroupInRequest, policy =>
            {
                policy.AuthenticationSchemes.Add(backOfficeAuthenticationScheme);
                policy.Requirements.Add(new UserGroupRequirement());
                policy.Requirements.Add(new UserGroupRequirement("userGroupIds"));
            });
```

Reply:
Both requirements must be met for it to succeed. When the query string doesn't match a value it returns true to satisfy the requirement but one of the requirements will match the query string.

Reply:
Have tested and this is working